### DISCUSSION TITLE
Pico VFS long terms question

### DISCUSSION THREAD

**Discussion by RoxerRN** [oyama/pico-vfs#52] [General]
Hi Oyama, first of all thank you very much for this project, it seems so promising to me. I am so interested by this functionality that's why I want to know I you plan to push this project to the pick-sdk or if it's more a toy code for you ? 

If you have big plan for pico-vfs I want to let you know that I am interested to help you with the development,  for example I think it can be usefull to add a way for user to use another external memory with litllefs but I am open to work on other more important feature if needed.

**Comment by oyama** [comment]
Hi RoxerRN,

Thank you for appreciating the project. From the beginning, pico-vfs has been intended to be integrated into the pico-sdk. I have already submitted a pull request to add it as a submodule to the pico-sdk:
https://github.com/raspberrypi/pico-sdk/pull/1715
However, I believe the pico-sdk team is currently very busy with the upcoming 1.6.0 release. My expectation is that 1.6.0 will be released around the end of summer, and I hope to discuss the integration with the pico-sdk team after that.

In the meantime, I wholeheartedly welcome contributions to improve the quality of pico-vfs. However, I am a bit cautious about feature expansions. While I have extensive experience with web and Unix technologies, I am a "beginner" in embedded technology, having a gap of about 30 years in this field. I acknowledge that my understanding of the current needs and user expectations for storage types in 2024 may be lacking.

Fortunately, pico-vfs has a modular structure, so I am very open to increasing the variety of supported storage options. I would love to discuss this in more detail and find specific ways we can collaborate effectively.

Thank you and best regards,
